src/components/Watchlists/Watchlist.module.scss

```diff
@@ -53,3 +57,9 @@
   align-items: center;
   width: 200px;
 }
+
+.subtitle {
+  @include text('body-1', 'm');
+
+  margin: 32px 16px 12px;
+}
```

Comment:
too big space between `Asset Overview` and `Categories` when recently is empty
![image](https://user-images.githubusercontent.com/24521041/60464669-eafc4580-9c57-11e9-8534-89f868705cfd.png)
maybe set `margin-bottom` for recently block

Reply:
![image](https://user-images.githubusercontent.com/25135650/60488693-4f94c000-9cab-11e9-8c7b-6ff9515b5750.png)